src/test/java/Homework17.java

```diff
@@ -5,26 +5,26 @@
 
 public class Homework17 extends BaseTest {
 
-    @Test
-    public void addSongToPlaylist() throws InterruptedException {
-        login("demo@class.com", "te$t$tudent");
-        Thread.sleep(2000);
-        grabASong();
-
-    }
-
-    public void grabASong(){
-        WebElement song = driver.findElement(By.xpath("//article[@data-test='song-card']"));
-        WebElement playlist = driver.findElement(By.xpath("//section[@id='playlists']//li[5]"));
-
-        Actions acts = new Actions(driver);
-        acts.clickAndHold(song)
-                .release(playlist)
-                .build()
-                .perform();
-
-        playlist.click();
-//        acts.dragAndDrop(song, playlist).build().perform();
-
-    }
+//    @Test
+//    public void addSongToPlaylist() throws InterruptedException {
+//        login("demo@class.com", "te$t$tudent");
+//        Thread.sleep(2000);
```

Comment:
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.